dijkstra.java

```diff
@@ -0,0 +1,67 @@
+import java.util.Arrays;
+
+public class DijkstraAlgorithm {
```

Comment:
Add description about this algo with link....
Also add comments in every function or wherever required.